File: KUPC2015/a.py
T = int(input())

for _ in range(T):
    S = input()
    flag = 0
    ans = 0
    for i, s in enumerate(S):
        if s == "o":
            flag += 1
            if flag == 2:
                if S[i-4:i+1] == "tokyo" or S[i-4:i+1] == "kyoto":
                    flag = 0
                    ans += 1
                else:
                    flag = 1
    print(ans)

File: KUPC2015/a.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
    int T;
    cin >> T;
    for (int t=0; t<T; ++t) {
        string S;
        cin >> S;
        int len;
        int ans = 0;
        len = S.size();
        for (int i=0; i<len-4; ++i) {
            string sub = S.substr(i, 5);
            if (sub == "tokyo" || sub == "kyoto") {
                ++ans;
                i += 4;
            }
        }
        cout << ans << endl;
    }
    return 0;
}